packages/in_app_purchase/lib/src/in_app_purchase/google_play_connection.dart

```diff
@@ -61,7 +61,11 @@ class GooglePlayConnection
     BillingResultWrapper billingResultWrapper =
```

Comment:
let's assert this parameter is null int app_store_connect so we ask developers to be more explicit about it. Which helps them to better understand that this is not required on iOS.

Reply:
Done

Reply:
I didn't see the diff of this change :(

Reply:
Can you pls elaborate more here. I have added the assert in **app_store_connection.dart** in **buyNonConsumable()**  https://github.com/rahulraj64/plugins/blob/88d5418fea54d1752f144077a36d2fa31df71b77/packages/in_app_purchase/lib/src/in_app_purchase/app_store_connection.dart#L59
But you are referring to the **google_play_connection.dart**

Reply:
Sorry I meant in `app_store_connect`.

Reply:
I saw it now! Thanks!